conformance/packages/dns-test/src/trust_anchor.rs

```diff
@@ -11,6 +11,28 @@ impl TrustAnchor {
         Self { keys: Vec::new() }
     }
 
+    pub fn public_dns() -> Self {
+        let does_not_matter = 3600;
```

Comment:
I would either inline this value as well or rename the variable. It's not clear to my why the `ttl` does not matter.

Reply:
we discovered recently that the TTL value is optional in trust anchor files (#2384) so the TTL value does not matter as DNS servers won't read it. we could add a new `DS` type with an optional TTL field to capture that detail here but it's important to be super accurate in this test code and that'd be out of scope for this PR.

I'll switch this to use the `DEFAULT_TTL` constant